src/markdown/mas/distance_sampling.Rmd

```diff
@@ -0,0 +1,4153 @@
+  select(regio, openheid, sbp, schatting = Estimate, se, lcl, ucl) %>%
+  arrange(regio, openheid, sbp) %>%
+  kable(digits = 3)
+```
+
+We visualiseren de densiteit per stratum.
+
+```{r}
+summary_results_veldleeuwerik_dist %>%
+  filter(variable == "density") %>%
+  mutate(stratum = paste(openheid, sbp, sep = " - ")) %>%
+  ggplot(aes(x = stratum, y = Estimate, colour = regio)) + 
+    geom_point(size = 4) + 
+    geom_errorbar(aes(ymin = lcl, ymax = ucl), width = 0.25) +
+    scale_y_continuous(limits = c(0, NA)) +
+    facet_wrap(~regio, scales = "free_y") +
+    labs(x = "", y = "Aantal broedparen per 100 ha", 
+         title = "Veldleeuwerik - densiteit 2022") +
+    theme(legend.position = "")
+```
+
+## Discussie {#discussie-distance2022}
+
+Net zoals voordien hadden we problemen om een passende detectiecurve te fitten (zie discussie \@ref(discussie-distance)).
+
+We zien een duidelijk verschil in densiteiten tussen de strata.
+Het is steeds hoger binnen dan buiten soortbeschermingszones en in OL dan in HOL.
+
+# Meedere soorten
+
+Hoe kunnen we deze workflow zo correct, efficiënt en snel toepassen op alle soorten?
+
+1.  Opruimen en voorbereiden basis dataframes met algemene data
+2.  Filter algemene dataframes
+3.  Genereer dataframes voor elke soort
+4.  Genereer dataframes voor distance sampling
+5.  Data selectie voor model specificatie
+6.  Model specificatie distance sampling
+
+Vanaf model selectie en fit zal geen of slechts beperkte automatisatie kunnen toegepast worden.
+We selecteren voor elke soort het finale model en slaan deze op.
+Deze finale modellen kunnen dan in het rapport gebruikt worden voor het genereren van plots en tabellen.
+
+## Opruimen en voorbereiden basis dataframes met algemene data
+
+We hebben 3 belangrijke dataframes.
+
+1.  `mas_clean` met alle observaties
+2.  `design_2018_2022_mas` met alle bezochte plots
+    -   selecteer OL in De Moeren en OL en HOL in Oostelijke leemstreek
+3.  `datum_grenzen_mas` met de datumgrenzen voor elke soort
+
+```{r}
+mas_clean <- mas_clean
+design_2018_2022_mas <- design_2018_2022_sf %>%
+  st_drop_geometry() %>%
+  filter((regio == "Oostelijke leemstreek" & openheid %in% c("OL", "HOL")) |
+         (regio == "De Moeren" & openheid == "OL"))
+datumgrenzen_mas <- datumgrenzen_mas 
+```
+
+## Filter algemene dataframes
+
+```{r}
+roofvogels <- c("Blauwe Kiekendief", "Boomvalk", "Bruine Kiekendief", "Buizerd", 
+                "Grauwe Kiekendief", "Ransuil", "Slechtvalk", "Sperwer", 
+                "Steenuil", "Steppekiekendief", "Torenvalk", "Velduil", 
+                "Wespendief", "Zwarte Wouw")
+```
+
+We maken een dataframe `mas_observations` waarbij we alle waarnemingen van vogels selecteren en filteren op broedcode \> 0, geen afstanden \> 300 m en dat ze binnen telperiodes vallen.
+We behouden ook enkel de plots die tot `design_2018_2022_mas` behoren.
+We voegen ook Haas toe en roofvogels (`r roofvogels`) zonder te filteren op broedcode.
```

Comment:
Na te vragen bij Johannes of de kraaiachtigen ook zoals de roofvogels behandeld moeten worden.

Reply:
Ok, zal ik doen